mne/realtime/fieldtrip_client.py

```diff
@@ -171,6 +171,8 @@ def _guess_measurement_info(self):
                     this_info['kind'] = FIFF.FIFFV_ECG_CH
                 elif ch.startswith('MISC'):
                     this_info['kind'] = FIFF.FIFFV_MISC_CH
+                elif ch.startswith('SYS'):
+                    this_info['kind'] = FIFF.FIFFV_SYST_CH
```

Comment:
But I'm trying to develop an interactive application that should just work, i.e. asking the user to record a small file is not so good.